### ISSUE TITLE
Ligatures Not Working

### ISSUE THREAD

**Issue by TheBotlyNoob** [kiliman/operator-mono-lig#166]
![image](https://user-images.githubusercontent.com/72417807/118556518-f95f8780-b731-11eb-8c6c-1783e32218e3.png)

And my font family

![image](https://user-images.githubusercontent.com/72417807/118556590-0bd9c100-b732-11eb-8c5e-23e2acb5919e.png)

**Comment by TheBotlyNoob** [comment]
Now noticing it didn't build `medium`

**Comment by kiliman** [comment]
I have a feeling that your SSm versions are not Mono.

Unfortunately we only have Book and Light for the non-SSm versions.

![image](https://user-images.githubusercontent.com/47168/118557425-295b5a80-b733-11eb-9439-212027e6b1f0.png)

**Comment by TheBotlyNoob** [comment]
Why doesn't `book` work?

**Comment by kiliman** [comment]
Which OS are you using?

Use `editor.fontWeight` to adjust weight instead of including in font family.

```json
"editor.fontFamily": "Operator Mono Lig",
"editor.fontLigatures": true,
"editor.fontWeight": "400",
```
Value | Common weight name
-- | --
100 | Thin (Hairline)
200 | Extra Light (Ultra Light)
300 | Light
400 | Normal (Regular)
500 | Medium
600 | Semi Bold (Demi Bold)
700 | Bold
800 | Extra Bold (Ultra Bold)
900 | Black (Heavy)
950 | Extra Black (Ultra Black)

</div><br class="Apple-interchange-newline">

**Comment by TheBotlyNoob** [comment]
My OS is Windows 10, and the font `Operator Mono Lig` doesn't seem to exist

**Comment by TheBotlyNoob** [comment]
hmmm, `Operator Mono Lig`, does exist, and works! but, it looks very weird

![image](https://user-images.githubusercontent.com/72417807/118671966-e5656580-b7c5-11eb-8a7a-07ccc53a187a.png)

**Comment by TheBotlyNoob** [comment]
upped the font size, still looks more like monospace than Operator Mono

![image](https://user-images.githubusercontent.com/72417807/118677253-1e073e00-b7ca-11eb-817e-7354b5f1c662.png)

**Comment by TheBotlyNoob** [comment]
if I look at it in the windows previewer it looks fine

![image](https://user-images.githubusercontent.com/72417807/118679125-9de1d800-b7cb-11eb-8011-3ec59aa7320e.png)

**Comment by kiliman** [comment]
Yeah, I just think that VS Code is using the Light weight. Try increasing fontWeight.

**Comment by TheBotlyNoob** [comment]
the `fontWeight` is at bold

**Comment by kiliman** [comment]
Are you using the numbers? Like "editor.fontWeight": "400"

Otherwise, I'm not sure what the problem is then.

**Comment by TheBotlyNoob** [comment]
I can try
